Gemfile

```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+gem "bundler-audit", github: 'rubysec/bundler-audit'
```

Comment:
I was looking into having this fix https://github.com/rubysec/bundler-audit/commit/4e32fca89d75f0e249671431ff38aadc02bfb28b
Which is not available on rubygems since the version was bumped 1 month earlier.